helm/dbs2go-global-r/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "dbs2go-global-r.fullname" . }}
+  labels:
+    {{- include "dbs2go-global-r.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+
+  ports:
```

Comment:
remove these values from values.yaml and put here values directly as these values will not be changed. This has been done for all other services as well.